sql/core/src/main/scala/org/apache/spark/sql/internal/SharedState.scala

```diff
@@ -43,23 +43,17 @@ private[sql] class SharedState(val sparkContext: SparkContext) extends Logging {
    */
   val listener: SQLListener = createListenerAndUI(sparkContext)
 
-  /**
-   * The base hadoop configuration which is shared among all spark sessions. It is based on the
-   * default hadoop configuration of Spark, with custom configurations inside `hive-site.xml`.
-   */
-  val hadoopConf: Configuration = {
```

Comment:
oh, I feel it is good to remove this extra layer since it does not really provide any benefit. I'd also like to avoid of calling addResource multiple times.